### DISCUSSION TITLE
Screen Stuck on loading

### DISCUSSION THREAD

**Discussion by dfurnell** [Blackymas/NSPanel_HA_Blueprint#1427] [Q&A]
Hi Guys, 

I am having some issues issues with my NS Panels stuck on the loading screen.

Today I have modified my yaml is ESP to match latest, updated the TFT and re-imported a clean blueprint?

The initializing screen shows as: 
TFT: 3.3
ESPHome: (Blank)
Blueprint: (Blank)

**Comment by edwardtfn** [comment]
What happens when you press the button Upload TFT on Home Assistant under Settings > Devices and Services > ESPHome?

Could you please share the ESPHome logs?

**Comment by edwardtfn** [comment]
Let's use #1429 for this.
